src/test/java/edu/ucsb/cs156/rec/integration/RequestTypeIT.java

```diff
@@ -0,0 +1,57 @@
+package edu.ucsb.cs156.rec.integration;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
```

Comment:
Not clear on the purpose of this file.   Integration Tests typically need to actually include a "test" of some kind, meaning a method with some assertion?

What *might* be helpful is a test that asserts that the database table exists... like creating an entity and storing it, and then asserting that it is in the database?   But even that seems a little premature.  I'd suggest just eliminating this file until you actually have some functionality to test.